ha/setup/ha_setup.py

```diff
@@ -858,6 +861,10 @@ def process(self):
                 self._confstore.delete(f"{const.PVTFQDN_TO_NODEID_KEY}/{node_name}")
             # Delete the config file
             self.remove_config_files()
+            if CleanupCmd.PRE_FACTORY_CHECK is True:
```

Comment:
I guess this check needs to be reverted:
If cleanup with pre-factory called: Revert things done in post_install command as well
else: do post_install after the current cleanup.

Reply:
Yes, corrected